doc/source/user_guide/make_container_image.rst

```diff
@@ -0,0 +1,99 @@
+.. _ref_make_container:
+
+Create Fluent Docker container image
+====================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procedure for creating a Fluent Docker container image uses a
+Linux machine to copy the needed files from the Fluent installation directory.
+
+
+Prerequisites
+-------------
+
+* A Linux machine with `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys license and account. Your Ansys reseller should have provided you with one.
+
+* The following provided files:
+  
+  * `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_
+  * `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_
+
+
+Procedure
+---------
+
+* If you have cloned `PyFluent <https://github.com/ansys/pyfluent>`_ locally then change the current working directory to
+`Docker files <https://github.com/ansys/pyfluent/blob/main/docker/fluent>`_ before executing these commands.
+
+* If you haven't cloned `PyFluent <https://github.com/ansys/pyfluent>`_ locally then copy `Docker files <https://github.com/ansys/pyfluent/blob/main/docker/fluent>`_ into an empty folder and
+execute these commands from that folder.
+
+Copy needed files
++++++++++++++++++
+
+Specify the pre-installed Ansys directory as a command line argument and run this script to copy needed files from the
+Ansys installation directory to the container image:
```

Comment:
Re "to the container image" aren't they copied to the current directory? The container image doesn't exist at this point.

Reply:
Yes. Done. Thank you.